src/main/java/org/gradlex/javaecosystem/capabilities/rules/CapabilityDefinitions.java

```diff
@@ -53,10 +28,15 @@
 import static org.gradlex.javaecosystem.capabilities.resolution.DefaultResolutionStrategy.NONE;
 
 public enum CapabilityDefinitions {
+
     AOPALLIANCE(HIGHEST_VERSION, AopallianceRule.class,
         "aopalliance:aopalliance",
         "org.springframework:spring-aop"
     ),
+    APACHE_CSV(HIGHEST_VERSION,
+            "org.apache.solr:solr-commons-csv",
+            "org.apache.commons:commons-csv"
```

Comment:
💄  Can you please do the indentation as the others.

I can have a look to generally change the formatting of the file, as you mentioned in the description. But if, we'll do that in a separate change later.

Reply:
I missed that, sorry